### PR TITLE
Update file paths, clean up script output a bit

### PR DESCRIPTION
**Pull request by jared-improbable**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -83,3 +83,5 @@ Scripts/spatialos.*.build.json
 
 # Don't ignore the Build util
 !SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/
+
+_ReSharper.caches
```
Comment (jared-improbable) [inline .gitignore]: Drive-by for people who've configured ReSharper to store caches within the working directory.

File: SpatialGDK/Extras/fastbuild/uninstall.ps1 [hunk 2]
```diff
@@ -36,14 +36,19 @@ if ($service) {
...
+    [System.IO.File]::Delete($brokerageFile)
+}
+
 Write-Host "Cleaning environment..."
 [System.Environment]::SetEnvironmentVariable("FASTBUILD_EXE_PATH", $null, [System.EnvironmentVariableTarget]::Machine)
 [System.Environment]::SetEnvironmentVariable("FASTBUILD_CACHE_PATH", $null, [System.EnvironmentVariableTarget]::Machine)
 [System.Environment]::SetEnvironmentVariable("FASTBUILD_BROKERAGE_PATH", $null, [System.EnvironmentVariableTarget]::Machine)
 [System.Environment]::SetEnvironmentVariable("FASTBUILD_CACHE_MODE", $null, [System.EnvironmentVariableTarget]::Machine)
 
-refreshenv
```
Comment (danielimprobable) [inline SpatialGDK/Extras/fastbuild/uninstall.ps1]: Why is refreshenv not needed anymore?

File: SpatialGDK/Extras/fastbuild/README.md [hunk 3]
```diff
@@ -14,13 +13,16 @@ FASTBuild can be installed in two different ways:
...
+* To install:
+* `powershell -NoProfile -ExecutionPolicy Bypass -File "\\lonv-file-01\Fastbuild\install.ps1" -service`
+* To uninstall:
+* `powershell -NoProfile -ExecutionPolicy Bypass -File "\\lonv-file-01\Fastbuild\uninstall.ps1" -service`
 
 # As a GUI
-* To install: `powershell -NoProfile -ExecutionPolicy Bypass -File "\\filesharing2\files\fastbuild\install.ps1"`
-* To uninstall: `powershell -NoProfile -ExecutionPolicy Bypass -File "\\filesharing2\files\fastbuild\uninstall.ps1"`
+* To install:
+* `powershell -NoProfile -ExecutionPolicy Bypass -File "\\lonv-file-01\Fastbuild\install.ps1"`
```
Comment (danielimprobable) [inline SpatialGDK/Extras/fastbuild/README.md]: Should this be indented rather than another bullet point?

File: SpatialGDK/Extras/fastbuild/uninstall.ps1 (hunk 2 above)
Comment (jared-improbable) [reply SpatialGDK/Extras/fastbuild/uninstall.ps1]: It never worked, and it's unlikely anyone will be trying to run fbuild commands from that powershell prompt.
